ui-tests/src/test/java/org/openmrs/reference/AddDiagnosisToVisitNoteTest.java

```diff
@@ -36,8 +34,6 @@ public void AddDiagnosisToVisitNoteTest() throws Exception {
         assertEquals("Pneumonia", visitNotePage.primaryDiagnosis());
         assertEquals("Bleeding", visitNotePage.secondaryDiagnosis());
         visitNotePage.save();
-        assertNotNull(patientDashboardPage.visitLink());
```

Comment:
Can you do some bit of explaining as to how this improves the reliability of this test?